src/moscot/problems/generic/_generic.py

```diff
@@ -62,7 +62,7 @@ def prepare(
         --------
         %(ex_prepare)s
         """
-        self.batch_key = key
+        self.batch_key = key  # type:ignore[misc]
```

Comment:
there are a lot of mypy type ignore it's a bit worrying tbh especially because they are scattered around the repo. Any idea on why would it be the case?

Reply:
resolved it by downgrading mypy again